tests/unit/test_runtime_build.py

```diff
@@ -24,31 +25,99 @@ def temp_dir(tmp_path_factory: TempPathFactory) -> str:
     return str(tmp_path_factory.mktemp('test_runtime_build'))
 
 
-def test_put_source_code_to_dir(temp_dir):
-    folder_name = _put_source_code_to_dir(temp_dir)
-
-    # assert there is a file called 'project.tar.gz' in the temp_dir
-    assert os.path.exists(os.path.join(temp_dir, 'project.tar.gz'))
-
-    # untar the file
-    with tarfile.open(os.path.join(temp_dir, 'project.tar.gz'), 'r:gz') as tar:
-        tar.extractall(path=temp_dir)
+def _check_source_code_in_dir(temp_dir):
```

Comment:
Can we please check if `agenthub` folder is in the directory?

Reply:
Sounds good! The sdist won't include `agenthub`, `evaluation`, and a lot of other directories. Just pushed a new commit to assert those.